app/assets/stylesheets/home.scss

```diff
@@ -251,60 +264,60 @@ a, a:hover {
 }
 
 .footer-heading {
-  font-size: 18px;
-  margin-bottom: 15px; }
+    font-size: 18px;
+    margin-bottom: 15px; }
 
 .footer-col-wrapper {
-  font-size: 15px;
-  margin-left: -15px; }
+    font-size: 15px;
+    margin-left: -15px; }
 
 .footer-col {
-  float: left;
-  margin-bottom: 15px;
-  padding-left: 15px; }
+    float: left;
+    margin-bottom: 15px;
+    padding-left: 15px; }
 
 .footer-col-1 {
-  width: -webkit-calc(20% - (30px / 2));
-  width: calc(20% - (30px / 2)); }
+    width: -webkit-calc(20% - (30px / 2));
+    width: calc(20% - (30px / 2)); }
 
 .footer-col-2 {
-  width: -webkit-calc(40% - (30px / 2));
-  width: calc(40% - (30px / 2)); }
+    width: -webkit-calc(40% - (30px / 2));
+    width: calc(40% - (30px / 2)); }
 
 .footer-col-3 {
-  width: -webkit-calc(40% - (30px / 2));
-  width: calc(40% - (30px / 2)); }
+    width: -webkit-calc(40% - (30px / 2));
+    width: calc(40% - (30px / 2)); }
 
 @media screen and (max-width: 800px) {
-  .footer-col-1,
-  .footer-col-2 {
-    width: -webkit-calc(50% - (30px / 2));
-    width: calc(50% - (30px / 2)); }
-
-  .footer-col-3 {
-    width: -webkit-calc(100% - (30px / 2));
-    width: calc(100% - (30px / 2)); } }
+    .footer-col-1,
+    .footer-col-2 {
+        width: -webkit-calc(50% - (30px / 2));
+        width: calc(50% - (30px / 2)); }
+
+    .footer-col-3 {
+        width: -webkit-calc(100% - (30px / 2));
+        width: calc(100% - (30px / 2)); } }
 @media screen and (max-width: 600px) {
-  .footer-col {
-    float: none;
-    width: -webkit-calc(100% - (30px / 2));
-    width: calc(100% - (30px / 2)); } }
+    .footer-col {
+        float: none;
+        width: -webkit-calc(100% - (30px / 2));
+        width: calc(100% - (30px / 2)); } }
 
 .page-article {
     background-color: rgb(246, 246, 239);
     border-bottom: 1px solid var(--ink-color);
     padding: 10px 0px 10px 20px;
-    
+
     .article-body {
         font-size: 12px;
         margin-left: 19px;
     }
-    
+
     .article-link {
         font-size: 12px;
         color: var(--ember-color);
     }
-    
+
```

Comment:
At 323 lines in your `home.scss`, this is getting pretty big-- big enough that GitHub doesn't want to show it to me until I expand it myself.

What do you think of splitting this up into logical units?

For example, any styles about the footer could go into `home/footer.scss`, and then import it into `home.scss` with `@import './home/footer';`

Reply:
That's a good idea!